apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingService.kt

```diff
@@ -17,4 +17,8 @@ class VedtaksvurderingService(
         logger.info("Henter vedtak for behandling med behandlingId=$behandlingId")
         return repository.hentVedtak(behandlingId)
     }
+
+    fun hentVedtakISak(sakId: Long): List<Vedtak> {
```

Comment:
nitpickq: `hentAlleVedtakISak`?

Reply:
jeg tenker det henger mer sammen med de andre funksjonene at denne ikke tar med alle, og ser ikke helt behovet for å endre de andre for å ha det likt